documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -42,6 +42,11 @@ It is forbidden to use both the `.indexing().enabled( )` and the deprecated `.in
 
 * The `auto-config` attribute is deprecated and will be removed in a future version.
 
+* Starting with version 11 it is mandatory to declare all indexed types in the indexing configuration or else warnings will be logged every time an undeclared type is indexed or queried.
+This requirement exists solely for indexed caches and applies to both Java classes and protobuf types.
```

Comment:
this will flood the logs for sure :smile:
I wonder if it's possible to print only at cache creation

Reply:
I'll try to move that to ```once```

Reply:
```@Once``` will be once for all caches I'm afraid. Ideally it should per once per cache

Reply:
I added an internal flag in ProtobufValueWrapperFieldBridge to log only once per cache